### PR TITLE
CI base image - Add Docker & Docker Compose

### PR DESCRIPTION
**Pull request by ondrejsika**

### CODE REVIEW

**Comment by ondrejsika** [comment]
Docker build is broken by this (no my patch, same error on branch develop)

```
 ---> Running in 8432c4df2452
pip 18.1 from /usr/lib/python3/dist-packages/pip (python 3.7)
Removing intermediate container 8432c4df2452
 ---> 6b9ee2ae1c0f
Step 35/37 : RUN trezorctl version
 ---> Running in e8449cf1a8a0
Traceback (most recent call last):
  File "/usr/local/bin/trezorctl", line 6, in <module>
    from trezorlib.cli.trezorctl import cli
  File "/usr/local/lib/python3.7/dist-packages/trezorlib/cli/trezorctl.py", line 30, in <module>
    from . import (
  File "/usr/local/lib/python3.7/dist-packages/trezorlib/cli/settings.py", line 19, in <module>
    from .. import device, firmware, messages, toif
  File "/usr/local/lib/python3.7/dist-packages/trezorlib/toif.py", line 5, in <module>
    import attr
ModuleNotFoundError: No module named 'attr'
The command '/bin/sh -c trezorctl version' returned a non-zero code: 1
```

After the rebase of its fix, it will pass.

**Comment by prusnak** [comment]
> Docker build is broken by this (no my patch, same error on branch develop)

This Python packages needs to be installed: https://pypi.org/project/attrs/ (via pip? `pip3 install attrs`)

**Review by mroz22 — APPROVED** [review]

File: package.json [hunk 1]
```diff
@@ -61,7 +61,6 @@
         "lerna": "^3.20.2",
         "npm-run-all": "^4.1.5",
         "patch-package": "^6.2.1",
-        "postinstall-postinstall": "^2.0.0",
```
Comment (keraf) [inline package.json]: Probably worth reverting that out of precaution now that the esbuild issue has been addressed (#3087)

Comment (mroz22) [reply package.json]: yeah, that was just experiments

**Review by keraf — APPROVED** [review]

**Comment by mroz22** [comment]
Not much here after merging all other related PR's.

**Review by szymonlesisz — APPROVED** [review]

**Pull request merged**